Airbnb.Application/Options/KafkaTopics.cs

```diff
@@ -0,0 +1,6 @@
+namespace Airbnb.Application.Options;
+
+public static class KafkaTopics
+{
+    public const string UserProfileUpdated = "UserProfileUpdated";
```

Comment:
Topic name should be Users and this is a Topic where User related events will be published and stored. We have a topic not per a single event, because many types of events can go to a single topic.